src/PowerShellEditorServices/Hosting/HostStartupInfo.cs

```diff
@@ -135,7 +135,7 @@ public sealed class HostStartupInfo
         /// <param name="currentUsersProfilePath">The path to the user specific profile.</param>
         /// <param name="featureFlags">Flags of features to enable.</param>
         /// <param name="additionalModules">Names or paths of additional modules to import.</param>
-        /// <param name="languageMode">The language mode inherited from the orginal PowerShell process. This will be used when creating runspaces so that we honor the same language mode.</param>
+        /// <param name="initialSessionState">The language mode inherited from the orginal PowerShell process. This will be used when creating runspaces so that we honor the same initialSessionState including allowed modules, cmdlets and language mode.</param>
```

Comment:
The initial part of this description is no longer accurate

Reply:
Oh dang, I thought I double-checked all of these. Ah well. For the documentation project.